kubernetes/config/kube_config.py

```diff
@@ -22,151 +22,243 @@
 from kubernetes.client import configuration
 from oauth2client.client import GoogleCredentials
 
-_temp_files = []
+from .incluster_config import ConfigException
```

Comment:
Good catch, fixed. (ConfigException was in incluster_config before, I moved it but apparently  in python you can steal other modules import, I don't understand the benefit of that :)